src/users.cc

```diff
@@ -42,6 +42,7 @@ static void user_name(char *ptr) {
     if (usr->ut_type == USER_PROCESS) {
       if (strlen(ptr) + strnlen(usr->ut_name, UT_NAMESIZE - 1) + 1 <= BUFLEN) {
         strncat(ptr, usr->ut_name, UT_NAMESIZE);
+        strncat(ptr, ", ", 2);
```

Comment:
You need to subtract the 2 extra characters from `UT_NAMESIZE` so this doesn't overflow.